src/handler.js

```diff
@@ -46,11 +45,25 @@ const responseWithEmptyBody = () => {
   return response
 }
 
-const isInvalidValidEvent = (event) => !('path' in event && 'httpMethod' in event)
+const isInvalidValidEvent = (event) => !(hasPath(event) && 'httpMethod' in event)
 
 const isGetMethod = (event) => event.httpMethod.toUpperCase() === 'GET'
 
-const isValidRequest = (event) => routes.includes(event.path) && isGetMethod(event)
+const hasPath = (event) => 'path' in event
+
+const getPath = (event) => {
```

Comment:
I wonder if this should be getPathString?

Reply:
Yes, seems a bit more intuitive